internal/livetiming/session.go

```diff
@@ -0,0 +1,67 @@
+package livetiming
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+	"time"
+)
+
+// Sessions contains all possible sessions in a two-depth map. The map is map[year]map[round]Session
+type Sessions map[int64]map[int64]Session
+
+type Session struct {
+	Season int64
```

Comment:
our convention is generally to just use `int` unless there's a reason to specify a sized `int64` here

Reply:
In this case the int64s were due to the `strconv.ParseInt` function returning `int64` instead of `int`, so I just left the keys as that type to avoid `int()` casts. If I swap to using `strconv.Atoi` like you suggested in the other comment, then this issue should resolve itself naturally.